ansible/tasks/lets_encrypt.yml

```diff
@@ -1,17 +1,29 @@
+# Sets up the let's encrypt certs only on first provision (the certbot steps cert step doesn't run if the cert exists)
+
+# Install certbot and pyth
```

Comment:
```suggestion
# Install certbot
```

but honestly, this comment can also be deleted (the task is self-explanatory).

Reply:
fixed in 88037ac